src/include/capacity.h

```diff
@@ -0,0 +1,74 @@
+/*-
+ * Copyright (c) 2014-2019 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+typedef enum {
+	WT_THROTTLE_CKPT,		/* Checkpoint throttle */
+	WT_THROTTLE_EVICT,		/* Eviction throttle */
+	WT_THROTTLE_LOG,		/* Logging throttle */
+	WT_THROTTLE_READ		/* Read throttle */
+} WT_THROTTLE_TYPE;
+
+#define	WT_THROTTLE_MIN			WT_MEGABYTE /* Config minimum size */
```

Comment:
Unexpected indentation, I'd suggest one <tab> before `WT_MEGABYTE` and one <tab> before the comment.

Reply:
Fixed.